docusaurus.config.ts

```diff
@@ -386,6 +386,10 @@ const config: Config = {
             from: "/nodes/deployments",
             to: "/nodes/networks",
           },
+          {
+            from: "/sdks/new-ts-sdk",
+            to: "/sdks/ts-sdk",
+          },
```

Comment:
@0xmaayan the redirect should be here https://deploy-preview-153--aptos-developer-docs.netlify.app/sdks/new-ts-sdk/